packages/api/src/controllers/access-control.ts

```diff
@@ -225,17 +225,22 @@ app.post(
 
     let config: Partial<GateConfig> = {};
 
-    if (user.viewerLimit) {
-      config.user_viewer_limit = user.viewerLimit;
-      config.user_id = user.id;
-    }
+    let viewerLimit = DEFAULT_MAX_CONCURRENT_VIEWERS;
 
-    if (user.createdAt > HACKER_DISABLE_CUTOFF_DATE) {
-      if (isFreeTierUser(user)) {
-        config.rate_limit = MAX_ALLOWED_VIEWERS_FOR_HACKER_TIER_PER_NODE;
+    if (user.stripeProductId) {
```

Comment:
What is stripe product id?

Reply:
It's the stripe `ID` of the product (plan) of the user. 
In this case either `Sandbox` (30 viewers as specified in config.js), `Growth` (50_000 viewers) and all the other untracked products (`Enterprise`, `Lotr`, ecc.).
If the user has no viewerLimit set in his object, it will fall back to product[user.productId].viewersLimit - if also that is missing, it will use the default const